README.md

```diff
@@ -1,7 +1,334 @@
+### Pushing a batch of documents
+For overall performance, it is better to push your documents in batches. Use the single document method when batches are not required to meet both your performance and volume requirements. For more information, visit [Managing Batches of Items in a Push Source](https://docs.coveo.com/en/90/cloud-v2-developers/managing-batches-of-items-in-a-push-source).
+
+```csharp
+PushDocument firstDocumentToAdd = new PushDocument("http://www.coveo.com/page1") {
+    ClickableUri = "http://www.coveo.com/page1",
+    ModifiedDate = DateTime.UtcNow
+};
+
+PushDocument secondDocumentToAdd = new PushDocument("http://www.coveo.com/page2") {
+    ClickableUri = "http://www.coveo.com/page2",
+    ModifiedDate = DateTime.UtcNow
+};
+
+IList<PushDocument> documentsToAdd = new List<PushDocument> {
+    firstDocumentToAdd,
+    secondDocumentToAdd
+};
+
+client.DocumentManager.AddOrUpdateDocuments(sourceId, documentsToAdd, null);
+```
+### Pushing a single document
+You should only use this method when you want to add or update a single document. Pushing several documents using this method may lead to the `429 - Too Many Requests` response from the Coveo platform and decrease the overall performance time of your application.
+
+```csharp
+string sourceId = "your Push API source ID";
+PushDocument document = new PushDocument("https://coveo.com") {
+    ClickableUri = "https://www.coveo.com",
+    ModifiedDate = DateTime.UtcNow
+};
+
+document.AddMetadata("title", "Coveo's home page.");
+
+PushDocumentHelper.SetContent(document, "this is Coveo's website home page.");
+
+client.DocumentManager.AddOrUpdateDocument(sourceId, document, null);
+```
+
+**Good to know:**
+* The `SetContent` method has an overload taking a Stream instead of a string.
+* The PushDocumentHelper class has also the method `SetContentFromFile` taking a file path as an argument.
+* The third argument in `AddOrUpdateDocument` is the ordering ID. If you don't provide a value, the SDK will create one using a timestamp to ensure the changes are performed in the order they were received. For more information about ordering ID, visit [Understanding the orderingId Parameter](https://docs.coveo.com/en/147/cloud-v2-developers/understanding-the-orderingid-parameter).
+* The call returns the generated ordering ID if you did not specify one. You can store it. It can be useful to delete a batch of documents.
+
+### Pushing a document with big properties
+In case the size of the document you just created is too big, the SDK does not handle it if you use the call to index a single document. If the whole document is larger than 256MB, it will throw an `ArgumentException`. If the document is larger than 5MB, it will throw a `CoveoPlatformException` including the status code (500) and `Internal server error` inside the `ErrorMessage` property. Take note that if you always use the batched call the SDK will handle it automatically.
```

Comment:
hmmm... What do you mean by "handle it" here? The error will still be thrown anyways, right?
With an answer to that question I'll be able to make suggestions on how to rephrase a few things here :)

Reply:
Si on utilise le call batcher, le SDK s'occupe de faire la logic nécessaire pour indexer le document, et donc pas d'erreur.

Sauf si le document est plus gros que 256mb, call batcher ou pas ca marche pas